src/components/TaskPicker.vue

```diff
@@ -182,11 +182,14 @@ const taskOptions = computed(() => {
 watch(
   () => props.inputVariants,
   (newVariants) => {
+    // @TODO: Fix this as it's not working as expected. When updating the data set in the parent component, the data is
```

Comment:
Can you point out how to replicate this? I went to create administration, and also update administration and it is not duplicating. TY

Reply:
To replicate this you need to disable `router.push({ path: APP_ROUTES.HOME });` on L#518 before updating the administration